src/components/seo.js

```diff
@@ -4,6 +4,22 @@ import Helmet from 'react-helmet'
 
 function SEO({ description, lang, meta, keywords, title }) {
   const metaDescription = description || 'default description'
+
+  const LocalBussinesSchemaOrg = {
+    '@context': 'http://schema.org',
+    '@type': 'LocalBusiness',
+    address: {
+      '@type': 'PostalAddress',
+      addressLocality: 'Poznań',
+      postalCode: '61-122',
+      streetAddress: 'Ostrówek 10',
+    },
+    name: 'ROBSON Bramy Garażowe',
+    openingHours: ['Mo-Fr 8:00-16:00'],
+    telephone: '(+48) 602 338 508',
```

Comment:
do you need these parentheses here?
```suggestion
    telephone: '+48 602 338 508',
```
or maybe even
```suggestion
    telephone: '+48602338508',
```
I don't know if these suggestions are correct, just make sure you follow the proper, recommended format.